tests/e2e/utils/s3_for_kserve/data_bucket.py

```diff
@@ -0,0 +1,73 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import sys, os, subprocess
```

Comment:
This file seems to be mostly copy/paste from https://github.com/awslabs/kubeflow-manifests/pull/527/files#diff-8e9fed5ddd7c7fc877d7e1906c6eaec901bb06d24e9d3d9735332b9283a4bc67 with the key difference being the file uploaded in this command. Maybe not in this pr, but we should catch unnecessary duplication like this if its possible. The create method here can take as an arg the cmd so this class can be reused.

`cmd = f"aws s3 sync ../../serving/tensorflow_serving/servables/tensorflow/testdata/saved_model_half_plus_two_cpu s3://{self.name}/saved_model_half_plus_two".split()`

Reply:
make tests/e2e/utils/s3_for_training/data_bucket.py generic by moving line 56-58 to fixture call